generator.yaml

```diff
@@ -13,18 +13,9 @@ operations:
       RetainAllVariantProperties: true
   DescribeModelPackage:
     custom_check_required_fields_missing_method: customCheckRequiredFieldsMissingMethod
-    primary_identifier_field_name: ARN
```

Comment:
I don't see this override in the new format. can you elaborate on how it works? because I see that model_package is_identifier has not changed

Reply:
Hey Suraj, here it is for your reference https://github.com/aws-controllers-k8s/code-generator/pull/190
The code it generates is the same but the way to enable it has changed. From setting `primary_identifier_field_name: ARN` to setting it as `is_arn_primary_key: true`

Reply:
Sg, I missed the diff b/w is_primary_key: true to is_arn_primary_key: true